Trimble.Modus.Components/Controls/TMSwitch/TMSwitchStyles.xaml

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ResourceDictionary xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                    xmlns:local="clr-namespace:Trimble.Modus.Components;assembly=Trimble.Modus.Components"
+                    x:Class="Trimble.Modus.Components.TMSwitchStyles">
+
+  <Style TargetType="local:TMSwitch" x:Key="SwitchStyle">
+    <Setter Property="TextColor" Value="{DynamicResource SwitchText}"/>
+    <Setter Property="VisualStateManager.VisualStateGroups">
+      <VisualStateGroupList>
+        <VisualStateGroup x:Name="CommonStates">
+          <VisualState x:Name="Selected">
```

Comment:
Switch should have a 'On' and 'Off' state. Default MAUI switch has those. Can you follow the same here?

https://learn.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/switch#switch-visual-states